src/main/java/com/sateekot/codereview/service/CodeReviewService.java

```diff
@@ -12,4 +12,11 @@ public String review() {
     logger.info("Reviewing code");
     return "Reviewing code";
   }
+
```

Comment:
Maintain proper order: static final fields should come before non-static fields.